apps/src/blockly/addons/cdoVariables.ts

```diff
@@ -46,3 +46,48 @@ export default function initializeVariables(
     }
   });
 }
+
+// Delete all variables except those that are in use in the workspace.
+export function deleteUnusedVariables(workspace: GoogleBlockly.Workspace) {
```

Comment:
This was moved out of `advancedProcedures.js` but there aren't any functional changes.